README.md

```diff
@@ -340,6 +340,9 @@ public interface ExampleHandler {
 }
 ```
 
+For this example, the generated class will respect `ExecutableHandler` interface
```

Comment:
I think it's better to write it as 'With this/such/aforementioned annotation..', as now it seems that it'll only work with the example.